### PR TITLE
NOISSUE - add dockerfile and change makefile

### PR DESCRIPTION
**Pull request by mteodor**
Signed-off-by: Mirko Teodorovic <mirko.teodorovic@gmail.com>

### CODE REVIEW

File: docker/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+WORKDIR /go/src/github.com/mainflux/agent
+COPY . .
+RUN apk update \
+    && apk add make\
+    && make $SVC \
+    && mv build/mainflux-$SVC /exe
+
+FROM scratch
+# Certificates are needed so that mailing util can work.
```
Comment (drasko) [inline docker/Dockerfile]: This comment is confusing, I can not see that certs are copied?

**Comment by drasko** [comment]
@mteodor DCO is failing, please fix that.

@blokovi please review.

**Comment by mteodor** [comment]
fixed

File: docker/Dockerfile (hunk 1 above)
Comment (drasko) [reply docker/Dockerfile]: Please review this comment.

**Pull request merged**